lib/game.ex

```diff
@@ -62,7 +80,20 @@ defmodule Game do
     writer.write(BoardPresenter.present(board))
   end
 
-  defp get_move(reader) do
-    Move.translate(reader.read)
+  defp get_input(reader) do
+    reader.read()
+    |> check_for_exit_command()
+  end
+
+  defp check_for_exit_command(input) do
+    if input == @exit_command do
+      {:error, :exit}
```

Comment:
I'm curious about the use of an `:error` tuple. The way I understand them is that they denote an error state, but would exiting the game be considered an error? I could be totally off base, but I'm curious!

Reply:
Yeah, I had the same thought when I added that, but I couldn't decide on another name. I'll think about what to change it to...